src/azure-cli/azure/cli/command_modules/cdn/custom/custom.py

```diff
@@ -64,6 +64,21 @@ def _update_mapper(existing, new, keys):
         setattr(new, key, new_value if new_value is not None else existing_value)
 
 
+def _convert_to_unified_delivery_rules(policy):
+    for existing_rule in policy.rules:
+        if existing_rule.conditions:
+            for con in existing_rule.conditions:
+                if con.parameters.odata_type == UrlPathMatchConditionParameters.odata_type and \
```

Comment:
How about changing it as follows to share `if con.parameters.operator is None and con.parameters.match_values is None:`?
```python
                if con.parameters.operator is None and con.parameters.match_values is None:
                   if con.parameters.odata_type == UrlPathMatchConditionParameters.odata_type:
                        con.parameters.operator = con.parameters.additional_properties["matchType"]
                        con.parameters.match_values = con.parameters.additional_properties["path"].split(',')
                   if con.parameters.odata_type == UrlFileExtensionMatchConditionParameters.odata_type:                  
                        con.parameters.operator = "Any"
                        con.parameters.match_values = con.parameters.additional_properties["extensions"]
```

Reply:
That make sense, thanks.